### PR TITLE
[Live/Auctions] Auction Information shows per-auction details about buyer's premum

### PR DESCRIPTION
**Pull request by orta**
@katarinabatina happy to change the wording on this button if you'd like?

shows_a_button_for_buyer_s_premium_when_needed 2x

Looks like this when you click:

screen shot 2016-05-23 at 4 24 02 pm

Not perfect, but could be made to look nicer in force pretty easily, which I can take a look at. I do need to make more of an effort on that side.

### CODE REVIEW

**Comment by orta** [comment]
Screenshots are a little different than reality ( we have a title for the Buyer's premium page, and there's a `'` in the button now )

File: Podfile.lock [hunk 1]
```diff
@@ -390,4 +390,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 0ad31c657e4ade9df73eedad2602ebd2abebebea
 
-COCOAPODS: 1.0.0
+COCOAPODS: 1.0.0.rc.1
```
Comment (ashfurrow) [inline Podfile.lock]: Changes here inadvertent?

Comment (orta) [inline Podfile.lock]: Yep

Comment (ashfurrow) [inline Podfile.lock]: Cool, looks great otherwise, feel free to merge.

**Comment by orta** [comment]
Cool beans 👍

**Pull request merged**